app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -30,13 +33,15 @@
 import javax.crypto.SecretKeyFactory;
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.PBEParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
 
 /**
  * An adapter to the default Java class for encrypting messages
  */
-@SuppressWarnings("checkstyle:ClassFanOutComplexity")
-// 1) Suppressed because the java crypto and security libraries require a lot of objects to use correctly
+@SuppressWarnings({"checkstyle:ClassFanOutComplexity", "PMD.ExcessiveImports"})
+// 1 & 2) Relies on many other classes for proper encryption, which is a trade of worth making
```

Comment:
https://www.merriam-webster.com/dictionary/trade-off but `tradeoff` is fine too according to wiki